diff --git a/node_modules/react-native-secure-storage/android/build.gradle b/node_modules/react-native-secure-storage/android/build.gradle
index fe7d65a..62df949 100644
--- a/node_modules/react-native-secure-storage/android/build.gradle
+++ b/node_modules/react-native-secure-storage/android/build.gradle
@@ -12,7 +12,7 @@ buildscript {
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 android {
     compileSdkVersion 26
